README.md

```diff
@@ -86,3 +86,8 @@ Registering the tag of loaded image as `latest` is available with below script.
 ```bash
 docker tag cuttlefish-orchestration:${PREBUILT_DOCKER_IMAGE_TAG} cuttlefish-orchestration:latest
 ```
+
+### Use docker image with Cloud Orchestrator
+
+Please refer to
+[here](https://github.com/google/cloud-android-orchestration/blob/main/docs/cloud_orchestrator.md).
```

Comment:
How about link to https://github.com/google/cloud-android-orchestration/blob/main/docs/docker.md ? Using docker at Cloud Orchestrator described here and `cloud_orchestrator.md` links to `docker.md` with docker scenario.

Reply:
Since `docs/cloud_orchestrator.md` describes the definition of CO and provides link toward `docs/docker.md`, I thought `docs/cloud_orchestrator.md` could be the better entry point.

But not a strong opinion. If you have opposite opinion, PLMK. I'll follow your decision then.

Reply:
I don't have strong opinion for that. Providing description about Cloud Orchestrator looks better than just providing docker image operations.

Thanks!